src/rpc/blockchain.cpp

```diff
@@ -799,16 +800,9 @@ static RPCHelpMan getblockfrompeer()
     const NodeContext& node = EnsureAnyNodeContext(request.context);
     ChainstateManager& chainman = EnsureChainman(node);
     PeerManager& peerman = EnsurePeerman(node);
-    CConnman& connman = EnsureConnman(node);
 
-    uint256 hash(ParseHashV(request.params[0], "hash"));
-
-    const NodeId nodeid = static_cast<NodeId>(request.params[1].get_int64());
-
-    // Check that the peer with nodeid exists
-    if (!connman.ForNode(nodeid, [](CNode* node) {return true;})) {
-        throw JSONRPCError(RPC_MISC_ERROR, strprintf("Peer nodeid %d does not exist", nodeid));
-    }
+    const uint256 hash(ParseHashV(request.params[0], "hash"));
```

Comment:
737cee7
```suggestion
    const uint256& hash{ParseHashV(request.params[0], "hash")};
```

Reply:
note to other reviewers: This is resolved in a later commit, could have been done here already probably.